DEV_GUIDE.md

```diff
@@ -297,10 +297,12 @@ has been applied ineffectively.
 ### Environment variables
 * `KROXYLICIOUS_IMAGE_REPO`: url to the image of kroxylicious to be used. Default value: `quay.io/kroxylicious/kroxylicious-developer`
 * `KROXYLICIOUS_VERSION`: version of kroxylicious to be used. Default value: `0.5.0-SNAPSHOT`
-* `KAFKA_VERSION`: kafka version to be used. Default value: `3.6.0`
+* `KAFKA_VERSION`: kafka version to be used. Default value: `3.6.1`
```

Comment:
hmm, this is going to rot whenever dependabot bumps kafka, bit of a pain

Reply:
The comment is misleading.  It default is the value discovered by determineKafkaVersion() from a value.  It is established by the build. KROXYLICIOUS_VERSION uses the same pattern.   We should reword the comment.

Reply:
Agree. Changed to indicate the value by default related to the value in the pom file